packages/liferay-npm-scripts/src/utils/spawnSync.js

```diff
@@ -22,27 +20,29 @@ function getDescription(command, args) {
  * @param {Object=} options={} Options to pass to spawn.sync
  */
 module.exports = function(command, args = [], options = {}) {
-	const {error, status} = spawn.sync(command, args, {
-		cwd: CWD,
-		env: {
-			...process.env,
-			PATH: `${PATH}:${process.env.PATH}`
-		},
+	const localCommand = path.join(
+		__dirname,
+		'../../node_modules/.bin',
```

Comment:
On Windows, `path.join()` will turn these "/" into "\\" as required. `__dirname` is absolute already, so we don't need `path.resolve()`.